libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -1,5 +1,5 @@
 <ng-container *ngIf="(readingList$ | async).length > 0; else empty">
-  <div class="reading-list-item" *ngFor="let b of readingList$ | async">
+  <div class="reading-list-item" data-testing="reading-list-item" *ngFor="let b of readingList$ | async">
     <div>
       <img class="reading-list-item--cover" [src]="b.coverUrl"  alt= "{{ b.title }}"/>
```

Comment:
When you fix task 1, I would advise you to rebase your branches, so that this small syntax issues (which belong to the code-review branch) get passed over!